libs/sdk-ui-theme-provider/src/tests/complementaryPalette.test.ts

```diff
@@ -0,0 +1,48 @@
+// (C) 2019-2020 GoodData Corporation
+import { IThemeComplementaryPalette } from "@gooddata/sdk-backend-spi";
+
+import { getComplementaryPalette } from "../complementaryPalette";
+
+describe("complementaryPalette", () => {
+    describe("getComplementaryPalette", () => {
+        const Scenarios: Array<[IThemeComplementaryPalette, IThemeComplementaryPalette]> = [
+            [
+                { shade0: "#001122", shade9: "#095f0f" },
+                {
+                    shade0: "#001122",
+                    shade1: "#01191f",
+                    shade2: "#02221d",
+                    shade3: "#032b1b",
+                    shade4: "#043319",
+                    shade5: "#053c17",
+                    shade6: "#064515",
+                    shade7: "#074d13",
+                    shade8: "#085611",
+                    shade9: "#095f0f",
+                },
+            ],
+            [
+                { shade0: "#001122", shade5: "#fcba03", shade9: "#095f0f" },
+                {
+                    shade0: "#001122",
+                    shade1: "#32321b",
+                    shade2: "#645415",
+                    shade3: "#97760f",
+                    shade4: "#c99809",
+                    shade5: "#fcba03",
+                    shade6: "#bfa306",
+                    shade7: "#828c09",
+                    shade8: "#45750c",
+                    shade9: "#095f0f",
+                },
```

Comment:
i suggest to switch this to snapshot test. let the computer do the 'typing' of the expected value for you.

Reply:
Good point, thanks.